### PR TITLE
Add elev.io in the docs (integration)

### PR DESCRIPTION
**Pull request by Go-Fred**

### CODE REVIEW

File: source/includes/integrations/_elevio.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## Elev.io
+
+[Elev.io](https://elev.io) delivers rich onsite help to customers. To enable Smooch inside elev.io, follow the guide below:
+
+### 1. Log in to elev.io and add Smooch's module
+
+After connecting to your elev.io dashboard, click on the menu "modules". Under the card named "Chat", click on "Install".
```
Comment (dannytranlx) [inline source/includes/integrations/_elevio.md]: Modules*

File: source/javascript/index.md [hunk 2]
```diff
@@ -43,6 +43,7 @@ includes:
   - integrations/stripe
   - integrations/stripe_notice_web
   - integrations/shopify
+  -  integrations/elevio
```
Comment (dannytranlx) [inline source/javascript/index.md]: nit: there's an extra space before `integrations`

**Comment by dannytranlx** [comment]
👍

File: source/includes/integrations/_elevio.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+![Elev.io Dashboard](/images/elevio_dashboard.png)
+
+Among the live chat solutions available on elev.io, choose Smooch.
+
+<img style="width:75%; min-width:400px; max-width:800px;" src="../images/elevio_chat.png" alt="Elev.io Chat">
+
+### 2. Connect to Smooch
+
+Paste your Smooch's app token in the appropriate text field.
```
Comment (mspensieri) [inline source/includes/integrations/_elevio.md]: > your Smooch app token

(remove 's)

File: source/includes/integrations/_elevio.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+<img style="width:75%; min-width:400px; max-width:800px;" src="../images/elevio_token.png" alt="Elev.io Token">
+
+### 3. Configure the widget
+
+Finally, configure the texts displayed on the widget and add the module to elev.io.
```
Comment (mspensieri) [inline source/includes/integrations/_elevio.md]: configure the text (no s)

**Comment by Go-Fred** [comment]
Done! Thanks for the reviews!

**Pull request merged**